public/app/percona/dbaas/components/DBCluster/__mocks__/DBCluster.hooks.ts

```diff
@@ -0,0 +1,13 @@
+import { Kubernetes } from 'app/percona/dbaas/components/Kubernetes/Kubernetes.types';
+import { dbClustersStub, getDBClustersActionStub } from './dbClustersStubs';
+import { DBCluster, GetDBClustersAction } from '../DBCluster.types';
+
+export const useDBClusters = (kubernetes: Kubernetes[]): [DBCluster[], GetDBClustersAction, boolean] => {
+  const dbClusters: DBCluster[] = [];
+
+  if (kubernetes.length > 0) {
+    dbClusters.push(...dbClustersStub);
+  }
+
+  return [dbClusters, getDBClustersActionStub, false];
```

Comment:
```suggestion
  return [[... dbClustersStub], getDBClustersActionStub, false];
```
And we could get rid of a bunch of code above )

Reply:
We want to return an empty array when there are no kubernetes, if we remove the rest of the code we won't cover that part.